### DISCUSSION TITLE
Dexcom Share not working for me post Dec 2 update

### DISCUSSION THREAD

**Discussion by samgray85** [Navid200/xDrip#146] [Dexcom]
Hi

I use Decom Share on Samsung S10. 

It stopped working along with everyone else last week

I installed the Dec 2 update, made no other changes. This is the error that results

![dexcomerror](https://user-images.githubusercontent.com/86637332/144786575-f76fcf64-8c6a-4f67-bcbf-f2ca759914a9.png)
![dexcomerror2](https://user-images.githubusercontent.com/86637332/144786589-de054323-590e-4ced-8414-1568fffe9510.png)
![dexcomerror3](https://user-images.githubusercontent.com/86637332/144786592-34091eb3-a2d5-42aa-bf63-64fec8f8a90c.png)
![dexcomerror4](https://user-images.githubusercontent.com/86637332/144786596-06cec767-aaec-423d-8e72-f644d81796b5.png)

**Comment by Navid200** [comment]
While you are editing your post, tap on the bottom border.  It let's you choose the image file to attach.
Please do that to attach an image instead of adding a link to an external file.

Please show a screenshot of the classic status page.

**Reply by samgray85** [reply]
Thanks @Navid200 - done (couldn't work it out before :)

**Reply by Navid200** [reply]
Great thanks.
I will ask for the right person to help.

**Comment by Navid200** [comment]
@samgray85 Would you please post in this issue and explain that you are still experiencing the problem even after having updated to the December 2 release?
https://github.com/NightscoutFoundation/xDrip/issues/1884

**Comment by Navid200** [comment]
You hadn't mentioned that your user ID was an email.

Please note that there are two issues.  

One is related to having an email address as a user ID:  https://github.com/NightscoutFoundation/xDrip/issues/1882
This has not been resolved yet.

The other was related to the delta value format having been changed from numbers to text.  This issue was resolved on December 2:  https://github.com/NightscoutFoundation/xDrip/issues/1884

Those two issues are not dependent on each other.  Just because one was resolved does not mean the other is too.
Please delete your post from the issue that was resolved on December 2 as your user ID is an email.

**Reply by samgray85** [reply]
OK - deleted the #1884 issue.
